app_base.htm

```diff
@@ -0,0 +1,56 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="menu_items">
+<ul class="nav navbar-nav">
```

Comment:
I think your navbar "li" items might look better if you made a list with the appropriate pages information and then just looped over 1 <li></li>

Reply:
I check your code, and I feel it is a very good idea to loop through a list to create li instead of doing my way. Thank you for the suggestion.